data/sql/updates/pending_db_world/rev_1609174873784850500.sql

```diff
@@ -0,0 +1,29 @@
+(17508, 0, 0, 1, 19, 0, 100, 0, 9579, 0, 0, 0, 0, 12, 17426, 8, 0, 0, 0, 0, 8, 0, 0, 0, 0, -2089.99, -11297.3, 63.522, 3.29479, 'Galaen\'s Corpse - On Quest \'Galaen\'s Fate\' Taken - Summon Creature \'Galaen\''),
+(17508, 0, 1, 2, 61, 0, 100, 0, 9579, 0, 0, 0, 0, 64, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen\'s Corpse - On Quest \'Galaen\'s Fate\' Taken - Store Targetlist'),
+(17508, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 9, 17426, 0, 0, 0, 0, 0, 0, 0, 'Galaen\'s Corpse - On Quest \'Galaen\'s Fate\' Taken - Send Target 1');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 17426);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17426, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 80, 1742600, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen - On Respawn - Run Script');
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` = 1742600);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1742600, 9, 0, 0, 0, 0, 100, 0, 2000, 2000, 0, 0, 0, 66, 0, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 0, 'Galaen - Actionlist - Set Orientation to Stored Target'),
+(1742600, 9, 1, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen - Actionlist - Say Line 0'),
+(1742600, 9, 2, 0, 0, 0, 100, 0, 4000, 4000, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen - Actionlist - Say Line 1'),
+(1742600, 9, 3, 0, 0, 0, 100, 0, 5000, 5000, 0, 0, 0, 1, 2, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen - Actionlist - Say Line 2'),
+(1742600, 9, 4, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 41, 5000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Galaen - Actionlist - Despawn In 5000 ms');
+
+-- Set correct quest prev for Galaen's Journal - The Fate of Vindicator Saruan based on WOW Head
+UPDATE `quest_template_addon` SET `PrevQuestID` = 9779 WHERE (`ID` = 9706);
```

Comment:
Well if you see this guide https://www.wow-pro.com/svens-bloodmist-leveling-guide-12-20-dutch/
it says
> In there, find Galaen’s Corpse (37,61) to turn in [18] Searching for Galaen and get the followup [18] Galaen’s Fate as well as [18] Galaen’s Journal – The Fate of Vindicator Saruan from Galaen’s Journal next to him.

This would only point to that Searching for Galaen should be the prerequisite.

Reply:
Hi, please check this video around min 18:
https://youtu.be/fmZFwMPIpR4
He did not complete Searching for Galaen yet but he can pick up the quest, also, look on the journal quest on wowhead, the comments agree on that being the pre req.